dist/pom.xml

```diff
@@ -19,6 +19,7 @@
 
     <properties>
         <genconf.dir>${project.build.directory}/generated-sources/conf</genconf.dir>
+        <version.metrics>3.2.6</version.metrics>
```

Comment:
in dist/pom.xml I left the old version of metrics-core and metrics-jetty8 because we don't have the 4.x version of metrics-jetty8